todo/src/App.js

```diff
@@ -1,28 +1,37 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import {connect} from 'react-redux'
 import './App.css';
+import TodoList from './components/TodoList';
+import Form from './components/Form'
 
 class App extends Component {
+  constructor(){
+    super();
+    this.state= {
```

Comment:
consider moving this state down into the the `<Form />` component since that's the only component concerned with this information